### ISSUE TITLE
Help menu accelerator (macOS only) not working

### ISSUE THREAD

**Issue by Ishaangandhi** [electron/electron#23683]
### Preflight Checklist

* [x] I have read the [Contributing Guidelines](https://github.com/electron/electron/blob/master/CONTRIBUTING.md) for this project.
* [x] I agree to follow the [Code of Conduct](https://github.com/electron/electron/blob/master/CODE_OF_CONDUCT.md) that this project adheres to.
* [x] I have searched the issue tracker for an issue that matches the one I want to file, without success.

### Issue Details

* **Electron Version:**
  * 8.2.5

* **Operating System:**
  * macOS 10.15.4

* **Last Known Working Electron version:**
  * Unknown

### Expected Behavior

Help menu role accelerator should open up the search item in the help menu.
### Actual Behavior

When typing in accelerator, nothing happens. 

### To Reproduce

Adding this as a menu:
    {
        label: 'Help',
        role: 'help',
        id: 'help',
        accelerator: 'CmdOrCtrl+Shift+p',
        submenu: [
            {
                label: 'Get support', }]}
Results in the help menu being shown, but the accelerator not doing anything.

**Comment by codebytere** [comment]
@Ishaangandhi macOS natively does not allow for accelerators to map to top-level MenuItems - this is not something within our control. There's a command one can use to bring focus to the toolbar:

<img width="780" alt="Screen Shot 2020-05-20 at 6 49 34 PM" src="https://user-images.githubusercontent.com/2036040/82514457-a9bc9080-9aca-11ea-857a-7407fa0e7573.png">

But this is something you yourself will have to recommend to users.

**Comment by Ishaangandhi** [comment]
Hi Shelley,
Thanks for the response. I think you are mistaken, however. One sanity check: try pressing Cmd+Shift+/ on Chrome, Calendar, or Notes. It's a very common shortcut, and is recommended for MacOS user interfaces.

**Comment by codebytere** [comment]
@Ishaangandhi i see how that might look like the top-level menu item - however, if you open the menu with a click just before pressing `Cmd+Shift+/` you'll see that it's actually resulting in a highlight of a menu item within the greater `View` menu - which _is_ supported functionality.

<img width="348" alt="Screen Shot 2020-05-20 at 7 21 55 PM" src="https://user-images.githubusercontent.com/2036040/82516272-2e111280-9acf-11ea-9804-5637bbaca889.png">

When a submenu item is triggered, it also results in highlighting of the top-level item which contains it.

**Comment by Ishaangandhi** [comment]
Ah I see the confusion. Let me rephrase my question for you: I want to be able to open the 'Search' field in the Help menu. How can I do that?

**Comment by codebytere** [comment]
@Ishaangandhi if i run the following: https://gist.github.com/cbbcfff6a3887ac19ed884b5a117a79b in [fiddle](https://github.com/electron/fiddle) and use the system shortcut (`Cmd+Shift+/`) it does work for me - i think what the issue is is that you're seeking to override it, which is what's not possible.

**Comment by Ishaangandhi** [comment]
Hm that's interesting. Using the system shortcut doesn't open the help menu for me, even without writing an accelerator.

**Comment by codebytere** [comment]
do you have this enabled?

<img width="780" alt="Screen Shot 2020-05-20 at 7 38 47 PM" src="https://user-images.githubusercontent.com/2036040/82517235-8e08b880-9ad1-11ea-9937-630cf4cd2830.png">

I had it on my mac by default, looks like

**Comment by Ishaangandhi** [comment]
I have it on in my mac, 
![image](https://user-images.githubusercontent.com/761037/82517453-295a5780-9aeb-11ea-918d-30c709f75b11.png)

It also works on quite a few apps, just not Electron.

**Comment by Ishaangandhi** [comment]
Interestingly enough, if I highlight another menu item, the accelerator will work, but besides that, it appears to do nothing.

**Comment by Ishaangandhi** [comment]
Is there a work around where I can intercept these keypresses and open up the search bar in the help menu? Inspecting the object, it appears not to have the search item in the items array.
